src/main/webapp/portal/main/partials/footer.html

```diff
@@ -12,7 +12,7 @@
 				<span><a ng-if='sessionCheckCtrl.back2ClassicURL' ng-href='{{sessionCheckCtrl.back2ClassicURL}}'>MyUW Classic</a></span>
 			</div>
 			<div>
-				<span>{{sessionCheckCtrl.user.version}}</span><span> - </span><span>{{sessionCheckCtrl.user.serverName}}</span><span> - </span><span>{{sessionCheckCtrl.user.sessionKey}}</span></span>
+				<span>{{sessionCheckCtrl.user.version}}</span><span ng-show="sessionCheckCtrl.user.serverName"> - </span><span>{{sessionCheckCtrl.user.serverName}}</span><span ng-show="sessionCheckCtrl.user.sessionKey"> - </span><span>{{sessionCheckCtrl.user.sessionKey}}</span></span>
```

Comment:
Agreed! Wasn't sure why it was there, I'll see if it works with this alternative.